service/src/test/java/com/ge/predix/controller/test/PolicyManagementControllerIT.java

```diff
@@ -81,13 +84,17 @@ public void setup() throws Exception {
         this.testZone2 = new TestUtils().createTestZone("PolicyMgmtControllerIT2");
         this.zoneService.upsertZone(this.testZone);
         this.zoneService.upsertZone(this.testZone2);
-        MockSecurityContext.mockSecurityContext(this.testZone);
-        MockAcsRequestContext.mockAcsRequestContext(this.testZone);
```

Comment:
Lines above (#81, 82, 84, 85) could be replaced with one call to TestUtils:
this.testZone = new TestUtils().setupTestZone(...).

No need to remove security and request contexts setup from here.  Only one test policyZoneDoesNotExistException() could overwrite security context with null zone and restore when done.

Setting up testZone2 could be deferred until the test that is using it -testCreateSamePolicyDifferentZones()